src/components/App.js

```diff
@@ -8,10 +8,10 @@ import { Route } from "react-router-dom"
 import PropTypes from "prop-types"
 
 const App = (props, { authUser }) => (
-    <React.Fragment>
+    <Fragment>
```

Comment:
Oh, wasn't aware you could do this. This will have to change for all instances of <React.Fragment>

Reply:
I can do this in this same PR